user.go

```diff
@@ -7,11 +7,7 @@ import (
 	"strings"
 )
 
-const (
-	UserIDDoesNotExist     string = "passage User with ID \"%v\" does not exist"
-	IdentifierDoesNotExist string = "passage User with Identifier \"%v\" does not exist"
-)
-
+// Deprecated: Use Get() instead.
```

Comment:
since `Get` isn't a direct replacement where this function is called, `Passage.GetUser -> Passage.User.Get`
```suggestion
// Deprecated: Use Passage.User.Get() instead.
```

same for the rest of the user functions

Reply:
made this change